DocumentationSite/articles/reflection.md

```
@@ -0,0 +1,762 @@
+
+typeof(Settings).TryGetStaticPropertyValue("Region", out var region);   // True
+```
+
+`TryGetStaticPropertyValue` is `BindingFlags.Static | BindingFlags.Public` — **public statics only**.
+A private static property returns `false` with a `null` value rather than throwing, which is easy to
+misread as "the property is null".
+
+The throwing overloads report through `InvalidOperationException`, not the `PropertyAccessException`
+family:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)